enemy.js

```diff
@@ -46,6 +46,7 @@ function createEnemy(src, width, height, i, maxHealth) {
   let enemy = {};
   enemy.element = document.createElement('img')
   enemy.element.id = 'enemy' + i;
+  enemy.element.classNmae = 'enemy'
```

Comment:
`classNmae` is misspelled.
It means `className` is never set, and is never used.
Delete this line.